### PR TITLE
Indicate which analyses are part of manuscript

### PR DESCRIPTION
**Pull request by sjspielman**
This PR addresses Issue #1287 and adds a column to `analyses/README.md` to indicate if the analysis module is part of the manuscript with `Yes` or `No`. Small phrasing changes were also made.

For reviewers: there are two modules I wasn't sure about (either because it's not decided yet or it wasn't clear to me), and these currently are marked "UNKNOWN." 
+ `rna-seq-composition`
+ `sample-distribution-analysis`

### CODE REVIEW

**Comment by jharenza** [comment]
>rna-seq-composition

This is not currently in the manuscript

>sample-distribution-analysis

I think this was subsumed by your plots of sample dist/seq strategy

File: analyses/README.md [hunk 1]
```diff
@@ -6,54 +6,55 @@ See the README of an individual analysis modules for more information about that
...
+| [`rna-seq-composition`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/rna-seq-composition) | `pbta-gene-expression-rsem-tpm.stranded.rds` <br> `pbta-histologies.tsv` <br> `pbta-mend-qc-results.tar.gz` <br> `pbta-mend-qc-manifest.tsv` <br> `pbta-star-log-manifest.tsv` <br> `pbta-star-log-final.tar.gz` | Analyzes the fraction of read types that comprise each RNA-Seq sample; flags samples with unusual composition| N/A | UNKNOWN | 
+| [`run-gistic`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/run-gistic) | `pbta-histologies.tsv` <br> `pbta-cnv-consensus.seg.gz` | Runs GISTIC 2.0 on SEG files | `pbta-cnv-consensus-gistic.zip` (included in data download) | Yes |
+| [`sample-distribution-analysis`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/sample-distribution-analysis) | `pbta-histologies.tsv` | Produces plots and tables that illustrate the distribution of different histologies in the PBTA data | N/A | UNKNOWN |
```
Comment (jaclyn-taroni) [inline analyses/README.md:50]: Agree that this was subsumed by something in `figures`.

```suggestion
| [`sample-distribution-analysis`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/sample-distribution-analysis) | `pbta-histologies.tsv` | Produces plots and tables that illustrate the distribution of different histologies in the PBTA data | N/A | No |
```

File: analyses/README.md [hunk 2]
```diff
@@ -6,54 +6,55 @@ See the README of an individual analysis modules for more information about that
...
+| [`oncoprint-landscape`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/oncoprint-landscape) | `pbta-snv-consensus-mutation.maf.tsv.gz` <br> `pbta-fusion-putative-oncogenic.tsv` <br> `analyses/focal-cn-file-preparation/results/consensus_seg_annotated_cn_autosomes.tsv.gz` <br> `analyses/focal-cn-file-preparation/results/consensus_seg_annotated_cn_x_and_y.tsv.gz` <br> `independent-specimens.*` | Combines mutation, copy number, and fusion data into an OncoPrint plot | N/A | Yes | 
+| [`rna-seq-composition`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/rna-seq-composition) | `pbta-gene-expression-rsem-tpm.stranded.rds` <br> `pbta-histologies.tsv` <br> `pbta-mend-qc-results.tar.gz` <br> `pbta-mend-qc-manifest.tsv` <br> `pbta-star-log-manifest.tsv` <br> `pbta-star-log-final.tar.gz` | Analyzes the fraction of read types that comprise each RNA-Seq sample; flags samples with unusual composition| N/A | UNKNOWN |
```
Comment (jaclyn-taroni) [inline analyses/README.md:48]:
```suggestion
| [`rna-seq-composition`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/rna-seq-composition) | `pbta-gene-expression-rsem-tpm.stranded.rds` <br> `pbta-histologies.tsv` <br> `pbta-mend-qc-results.tar.gz` <br> `pbta-mend-qc-manifest.tsv` <br> `pbta-star-log-manifest.tsv` <br> `pbta-star-log-final.tar.gz` | Analyzes the fraction of read types that comprise each RNA-Seq sample; flags samples with unusual composition| N/A | No | 
```

**Review by jaclyn-taroni — APPROVED** [review]
Agree with @jharenza's statements. I will commit those changes, which is why I am approving!

**Comment by jharenza** [comment]
Just double checked this in the manuscript repo. We had added MENDQC in our quality control section, though we do not have any cutoffs for use/not use of data:

https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/content/06.methods.md#quality-control-of-sequencing-data

>MEND QC [[42](https://alexslemonade.github.io/OpenPBTA-manuscript/#ref-xCe0Gxab)] was performed on aligned RNA-Seq reads using [this workflow](https://github.com/d3b-center/publication_workflows/blob/master/openPBTA/kfdrc-mendqc-wf.cwl) to identify mapped exonic non-duplicate reads.

So, should we include that here or remove from the QC section?

**Comment by jaclyn-taroni** [comment]
I would remove it from the QC section if we did not use it and leave this as is

**Pull request merged**